SYCL/ESIMD/dpas/dpas_int.cpp

```diff
@@ -0,0 +1,58 @@
+//==---------------- dpas_int.cpp  - DPC++ ESIMD on-device test ------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc || esimd_emulator
```

Comment:
For follow-up in later PR: dg2 should also work, right?

Reply:
There will be more tests with ExecutionSize=8 (testing ACM/dg2)